code/client/launcher/CitizenGame.cpp

```diff
@@ -571,7 +577,10 @@ void CitizenGame::Launch(const std::wstring& gamePath, bool isMainGame)
 			if (getenv("CitizenFX_ToolMode"))
 			{
 				// pre-load the correct chrome_elf.dll
-				LoadLibraryW(MapRedirectedFilename(L"Social Club/chrome_elf.dll").c_str());
+				HMODULE chromeElfHandle = LoadLibraryW(MapRedirectedFilename(L"Social Club/chrome_elf.dll").c_str());
+
+				if(!chromeElfHandle)
+					return (HMODULE)INVALID_HANDLE_VALUE;

```

Comment:
chrome_elf is again a disambiguation, failing to load `libcef.dll` however might lead to confusing errors instead.